package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "engines": {
     "node": ">=16 <19",
```

Comment:
You should also support Node LTS which is v20 now, so bump it to <21.

Reply:
I would, except we have a dependency that doesn't support v20 yet and we need an update for that first.